src/GitHub.App/SampleData/SampleViewModels.cs

```diff
@@ -25,6 +25,7 @@ namespace GitHub.SampleData
     public class BaseViewModelDesigner : ReactiveObject, IViewModel
     {
         public ICommand Cancel { get; set; }
+        public ReactiveCommand<object> CancelCommand { get; protected set; }
```

Comment:
Let's kill everything in this designer and inherit it from `BaseViewModel` instead, which already has everything defined.